src/extensions/multiple-sort/bootstrap-table-multiple-sort.js

```diff
@@ -712,3 +700,21 @@ BootstrapTable.prototype.setButtonStates = function () {
     this.$sortModal.find('#delete').attr('disabled', 'disabled')
   }
 }
+
+BootstrapTable.prototype.multiSort = function (sortPriority) {
+  this.options.sortPriority = sortPriority
+  this.options.sortName = ''
+
+  if (this.options.sidePagination === 'server') {
+    const t = this.options.queryParams
```

Comment:
Can this variable be named as queryParams?

Reply:
This part of the code was just moved to a function (not written by me), also this extension shoud be rewritten sometime.
But i removed the variable and use the queryParams without a variable.